drivers/sensor/pmw3360/Kconfig

```diff
@@ -0,0 +1,80 @@
+# Kconfig - Sensor data simulator
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig PMW3360
+	bool "PMW3360 mouse optical sensor"
+	help
+	  Enable PMW3360 mouse optical sensor.
+
+if PMW3360
+
+config PMW3360_DEV_NAME
+	string "PMW3360 device name"
+	default "PMW3360"
+	help
+	  Device name for PMW3360.
+
+config PMW3360_CS_GPIO_DEV_NAME
+	string "PMW3360 CS GPIO device"
+	default "GPIO_0"
+	help
+	  Device to which CS pin is connected.
+
+config PMW3360_CS_GPIO_PIN
```

Comment:
Such settings should be configured via device tree. Is it still impossible to add dts bindings in the `-nrf` repo?

Reply:
I have no idea - it was suppose to but I am not sure anybody is using it...
Can this be done later? This is not the only case and I would like to have it in the repo to have project given out earlier. We will still have chance to do it as there will be another sensor driver added - we can add bindings there.

Reply:
Agreed. Let's leave it as a future improvement.